### ISSUE TITLE
Missing Organism for GSE54716

### ISSUE THREAD

**Issue by Miserlou** [AlexsLemonade/refinebio#833]
https://api.refine.bio/experiments/GSE54716/

`organisms` is `[]`, which shouldn't happen. Not a problem for GSE54717.

**Comment by kurtwheeler** [comment]
I just tested this accession locally and it was properly associated with the correct organism. I wonder if this is a bug that has already been fixed?

As far as fixing this data goes though... I propose two steps:
  1. Do a query to figure out if there's any other experiments that had the same problem.
  2. Unsurvey those experiments and re-survey them in order to re-collect the metadata and fix the data.

@Miserlou does that sound like a good plan to you?

**Comment by kurtwheeler** [comment]
I've found 129 total experiments without organisms:
```
  id  | accession_code 
------+----------------
  183 | E-MEXP-2381
 1560 | GSE106981
 2380 | GSE33539
  340 | GSE37164
 2799 | GSE49675
 1329 | GSE22337
 2075 | GSE42373
 1469 | GSE13033
 2448 | GSE44597
  497 | GSE34567
 1883 | GSE53537
  696 | GSE38123
 2707 | GSE59795
 2826 | GSE41498
 1680 | GSE107417
 2463 | GSE37044
 2465 | GSE23973
 1716 | GSE12999
 1772 | GSE12538
 1785 | GSE103171
 1805 | GSE22317
   72 | E-MEXP-3893
  136 | E-TABM-748
  200 | E-MEXP-2215
  285 | E-MTAB-1476
  279 | GSE31186
  300 | E-MEXP-2288
  430 | GSE37163
  415 | GSE31185
  613 | GSE55881
  472 | GSE13157
  531 | GSE45635
  510 | GSE23203
  539 | GSE36743
  560 | GSE34886
  706 | GSE17818
  661 | GSE17816
  646 | GSE54716
  676 | GSE32298
  675 | GSE35417
  693 | GSE27382
  738 | GSE64888
 2007 | GSE44808
 2484 | GSE32666
 2061 | GSE31229
 2138 | GSE37045
 2168 | GSE33112
 2498 | GSE51835
 2201 | GSE57750
 2259 | GSE43114
 2277 | GSE41914
 2293 | GSE40624
 2299 | GSE34776
 2312 | GSE4218
 2414 | GSE37371
 2578 | GSE54378
 2611 | GSE54762
 2631 | GSE48501
 2633 | GSE60124
 3019 | GSE49695
 3110 | GSE49586
 3162 | GSE47934
 3165 | GSE51395
 3523 | GSE51928
 3561 | GSE4217
 3593 | GSE31864
 3648 | GSE56645
 3637 | GSE33050
 3621 | GSE8302
 3644 | GSE33167
 3781 | GSE42259
 3757 | GSE40122
 3755 | GSE33113
 3777 | GSE39907
 3769 | GSE61928
 3788 | GSE39504
 3771 | GSE40496
 3808 | GSE40830
 3810 | GSE54314
 3811 | GSE32727
 3817 | GSE37253
 3825 | GSE43591
 3841 | GSE41364
 3858 | GSE43116
 3865 | GSE56315
 3874 | GSE56967
 3872 | GSE57520
 3873 | GSE45022
 3893 | GSE6550
 3899 | GSE9974
 3900 | GSE74608
 3959 | GSE6872
 3971 | GSE67508
 4000 | GSE79490
 4052 | GSE33010
 4080 | GSE79003
 4072 | GSE74664
 4137 | GSE62753
 4139 | GSE43115
 4182 | GSE58800
 4167 | GSE39452
 4147 | GSE20357
 4186 | GSE6972
 4196 | GSE52037
 4217 | GSE73091
 4227 | GSE62649
 4241 | GSE68694
 4261 | GSE3045
 4272 | GSE8140
 4342 | GSE49584
 4336 | GSE43177
 4345 | GSE33727
 4358 | GSE23968
 4361 | GSE36766
 4333 | GSE44841
 4380 | GSE38597
 4369 | GSE34205
 4414 | GSE66162
 4396 | GSE39720
 4405 | GSE40985
 4439 | GSE7832
 4475 | GSE43068
 4534 | GSE35956
 4523 | GSE31246
 4558 | GSE72747
 4544 | GSE55284
 4574 | GSE81870
 4587 | GSE59229
 4604 | GSE58049
(129 rows)
```

**Comment by kurtwheeler** [comment]
The list is growing:
```
data_refinery=> select count(*) from experiments where organism_names = '{}';
 count 
-------
  1300
(1 row)
```

**Comment by Miserlou** [comment]
That does seem to indicate that this wasn't magically fixed as we had hoped, and I don't immediately see any reason why this should happen from looking at a sample of those accessions. Maybe it's being un-set elsewhere in the process rather than not being set at all?

**Comment by kurtwheeler** [comment]
Hmm that's a good thought, I'll consider that as I dig into this.

**Comment by cgreene** [comment]
Closed by #1119
